lib/resty/limit/req.lua

```diff
@@ -150,6 +193,14 @@ function _M.limit(cfg)
     local scale = 1
     local len = #rate
 
+    if not conn_master then
```

Comment:
hm, i think it's added back to connection pool every time despite of this. https://github.com/hostinger/lua-resty-limit-req/blob/1bc36922a00640c547eb455925d8ec5af71613c4/lib/resty/limit/req.lua#L166-L169